opentelemetry-sdk/src/metrics/internal/aggregate.rs

```diff
@@ -127,7 +128,12 @@ impl<T: Number<T>> AggregateBuilder<T> {
                 };
                 let g = g.unwrap_or_else(|| new_agg.as_mut().expect("present if g is none"));
 
-                lv_agg.compute_aggregation(&mut g.data_points);
+                match t {
+                    Some(Temporality::Delta) => {
+                        lv_agg.compute_aggregation_delta(&mut g.data_points)
```

Comment:
nit - how about naming these methods delta() and cumulative(), to be consistent with naming in other aggregates?

Reply:
The code setup for `last_value` is not consistent with other aggregates. `delta` and `cumulative` of other aggregates return a tuple `(usize, Option<Box<dyn Aggregation>>)` whereas the `compute_aggregation` methods of `last_value` do not return anything.

I would like to make the code setup for `Gauge` consistent with other aggregates in a different PR. Is it okay to fix it then?